test/e2e/run_dns_test.go

```diff
@@ -60,7 +60,7 @@ var _ = Describe("Podman run dns", func() {
 		session := podmanTest.Podman([]string{"run", "--hostname=foobar", ALPINE, "cat", "/etc/hostname"})
 		session.WaitWithDefaultTimeout()
 		Expect(session).Should(ExitCleanly())
-		Expect(session.OutputToString()).To(Equal("foobar"))
+		Expect(session.OutputToString()).To(Equal("foobar\n"))
```

Comment:
please use the exact diff I posted this will not work like that because OutputToString() will trim the newline `Expect(string(session.Out.Contents())).To(Equal("foobar\n"))`

Reply:
got it